dask_expr/io/json.py

```diff
@@ -114,7 +113,6 @@ def read_json(
         path_converter=path_converter,
         **kwargs,
     )
-    return from_legacy_dataframe(df)
```

Comment:
Nice. It looks like we can do this now since the changes over in https://github.com/dask/dask/pull/11529 make some methods (e.g. `dd.read_json`) more `dask-expr` aware

Reply:
Yep